binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/pom.xml

```diff
@@ -84,6 +84,43 @@
 			<artifactId>awaitility</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.micrometer</groupId>
+			<artifactId>context-propagation</artifactId>
```

Comment:
I think this one is supposed to be `optional`.
Otherwise it is going to cause some undesired fluctuations with those `contextWrite()` and brothers in those cases when target project is not interested in the observation.